internal/services/mssql/mssql_server_resource_test.go

```diff
@@ -305,6 +305,47 @@ func TestAccMsSqlServer_CMKServerTagsUpdate(t *testing.T) {
 	})
 }
 
+func TestAccMsSqlServer_writeOnlyAdminLoginPassword(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_mssql_server", "test")
+	r := MsSqlServerResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.writeOnlyAdminLoginPassword(data, "7h1515K4711-secret", 1),
+			Check:  check.That(data.ResourceName).ExistsInAzure(r),
+		},
+		data.ImportStep("administrator_login_password_wo_version"),
+		{
+			Config: r.writeOnlyAdminLoginPassword(data, "7h1515K4711-updated", 2),
+			Check:  check.That(data.ResourceName).ExistsInAzure(r),
+		},
+		data.ImportStep("administrator_login_password_wo_version"),
+	})
+}
+
+func TestAccMsSqlServer_updateToWriteOnlyPassword(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_mssql_server", "test")
+	r := MsSqlServerResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check:  check.That(data.ResourceName).ExistsInAzure(r),
+		},
+		data.ImportStep("administrator_login_password"),
+		{
+			Config: r.writeOnlyAdminLoginPassword(data, "7h1515K4711-secret", 1),
+			Check:  check.That(data.ResourceName).ExistsInAzure(r),
+		},
+		data.ImportStep("administrator_login_password", "administrator_login_password_wo_version"),
+		{
+			Config: r.basic(data),
+			Check:  check.That(data.ResourceName).ExistsInAzure(r),
+		},
+		data.ImportStep("administrator_login_password"),
+	})
+}
```

Comment:
These tests need to be skipped for Terraform versions < 1.11. There's an example in the preliminary guide I shared on how to do that, or you can take a look at the tests over in [my PR](https://github.com/hashicorp/terraform-provider-azurerm/blob/926de434159b8fbfad1a9092fc4aa6f3ee1bfb5c/internal/services/mysql/mysql_flexible_server_resource_test.go#L518-L569).

Reply:
Oh shoot! Swapped! Thank you thank you